docs/api-docs/cart-and-checkout/checkout-sdk.md

```diff
@@ -46,8 +46,8 @@ The Checkout JS SDK allows you to present the checkout process to the shopper in
 However, the Checkout JS SDK does not allow you to change the checkout's underpinnings   - you must still conform to the model of the Checkout API underpinning the SDK to complete a Checkout and create an Order in BigCommerce.
 The SDK does not allow you to implement custom payment, shipping, or tax calculation providers into the checkout. Instead, you can use the configured providers for these services on a given store via our Checkout API.
 
-### Paypal credit card options
-A universal PayPal payment method, simply called Paypal, substitutes the **Place Order** button with a PayPal smart button on your checkout page. This Paypal method requires checkout SDK users who build custom checkouts to provide additional initialization options. For more details on adjusting your code to make it work with the PayPal smart button, see the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paypalcommercepaymentinitializeoptions.md).
+### <a name="required-payment-method-initialization-options"> </a>Required payment method initialization options 
```

Comment:
What is the purpose of this `<a>` tag? Is this being linked to from somewhere else?

Reply:
I created an anchor tag.